package.json

```diff
@@ -110,7 +118,7 @@
     "debug": "tsx src/cli.ts",
     "lint": "alex . && knip && npm run lint:ts && npm run prettier && npm run schemas:check",
     "lint:ts": "eslint . --ext .js,.ts",
-    "prebuild": "rm -rf dist/",
+    "prebuild": "rm -rf dist/ && rm -f src/package.json && ln package.json src/package.json",
```

Comment:
Ran into a small problem with this where upon first cloning down the branch I needed to set up this `package.json` link before being able to run `./bin/dev.js`. Might want to add a note about it to the CONTRIBUTING file.

Reply:
ah, that's a great callout ty

Reply:
https://github.com/readmeio/rdme/pull/1068/commits/e990a2bbda5ac30b78fba890256d435474853534